src/kernel/lsm/compactor.rs

```diff
@@ -132,29 +120,22 @@ impl Compactor {
                 let ss_table_futures = vec_sharding.into_iter()
                     .map(|(gen, sharding)| {
                         async move {
-                            SSTable::create_for_mem_table(
-                                self.config(),
-                                gen,
-                                self.sst_factory(),
-                                sharding,
-                                level + 1,
-                                IoType::Direct
-                            )
+                            self.ver_status()
+                                .loader()
+                                .create(gen, sharding, level + 1)
                         }
                     });
-                let vec_ss_table_and_scope: Vec<(SSTable, Scope)> = future::try_join_all(ss_table_futures).await?;
-                let (vec_new_ss_table, vec_new_scope): (Vec<SSTable>, Vec<Scope>) = vec_ss_table_and_scope
+                let vec_ss_table_and_scope: Vec<(Scope, SSTableMeta)> = future::try_join_all(ss_table_futures).await?;
```

Comment:
why SSTable change to SSTableMeta?

Reply:
Because the SSTable here should be controlled by the SSTableLoader to avoid abuse, and the original logic here is just to insert the SSTable into the SSTableLoader, and get the meta in the middle, which is directly simplified here

Reply:
get